frontend/admin/src/js/components/poolCandidate/ViewPoolCandidate/ViewPoolCandidatePage.tsx

```diff
@@ -0,0 +1,237 @@
+import { getLocalizedName } from "@common/helpers/localize";
+import UserProfile from "@common/components/UserProfile";
+import { useAdminRoutes } from "admin/src/js/adminRoutes";
+import { useState } from "react";
+import { Applicant } from "@common/api/generated";
+import {
+  Scalars,
+  useGetPoolCandidateSnapshotQuery,
+  PoolCandidate,
+  Maybe,
+} from "../../../api/generated";
+import DashboardContentContainer from "../../DashboardContentContainer";
+
+export interface ViewPoolCandidateProps {
+  poolCandidate: PoolCandidate;
+}
+
+type SpacerProps = React.HTMLProps<HTMLDivElement>;
+
+const Spacer = ({ children, ...rest }: SpacerProps) => (
+  <div data-h2-padding-bottom="base(1.5rem)" {...rest}>
+    {children}
+  </div>
+);
+
+export const ViewPoolCandidate = ({
+  poolCandidate,
+}: ViewPoolCandidateProps): JSX.Element => {
+  const intl = useIntl();
+  const adminPaths = useAdminRoutes();
+
+  // prefer the rich view if available
+  const [preferRichView, setPreferRichView] = useState(true);
+
+  const links = [
+    {
+      title: intl.formatMessage({
+        defaultMessage: "My pools",
+        id: "0Y4Dt4",
+        description: "Breadcrumb title for the pools page link.",
+      }),
+      href: adminPaths.poolTable(),
+    },
+    {
+      title: getLocalizedName(poolCandidate.pool.name, intl),
+      href: adminPaths.poolView(poolCandidate.pool.id),
+    },
+    {
+      title: intl.formatMessage({
+        defaultMessage: "All candidates",
+        id: "e9FNqp",
+        description: "Breadcrumb title for the All Candidates page link.",
+      }),
+      href: adminPaths.poolCandidateTable(poolCandidate.pool.id),
+    },
+    {
+      title: `${poolCandidate.user.firstName} ${poolCandidate.user.lastName}`,
+    },
+  ] as BreadcrumbsProps["links"];
+
+  const parsedSnapshot: Maybe<Applicant> = JSON.parse(
+    poolCandidate.profileSnapshot,
+  );
+  const snapshotUserPropertyExists = !!parsedSnapshot;
+
+  let mainContent: React.ReactNode;
+  if (snapshotUserPropertyExists && preferRichView) {
+    mainContent = (
+      <UserProfile
+        applicant={parsedSnapshot}
+        sections={{
+          myStatus: { isVisible: false },
+          hiringPools: { isVisible: false },
+          about: { isVisible: true },
+          language: {
+            isVisible: true,
+          },
+          government: {
+            isVisible: true,
+          },
+          workLocation: {
+            isVisible: true,
+          },
+          workPreferences: {
+            isVisible: true,
+          },
+          employmentEquity: {
+            isVisible: true,
+          },
+          roleSalary: { isVisible: true },
+          skillsExperience: {
+            isVisible: true,
+          },
+        }}
+        isNavigationVisible={false}
+      />
+    );
+  } else if (snapshotUserPropertyExists && !preferRichView) {
+    mainContent = <pre>{JSON.stringify(parsedSnapshot, null, 2)}</pre>;
```

Comment:
Could we put some overflow scroll on this to prevent longer lines from causing odd background colour issues.

<img width="285" alt="Screen Shot 2022-09-16 at 11 53 18 AM" src="https://user-images.githubusercontent.com/4127998/190679892-4673bee9-ebf5-40df-9771-8f5faedfd705.png">

Reply:
Good idea!  I added it to [Add x scroll to json view](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3982/commits/f581420ad302bbf6fa6373e53ed9024212c4ecfd).